client/packages/system/src/Immunisation/VaccineCourseEditModal/VaccineCourseEditModal.tsx

```diff
@@ -0,0 +1,324 @@
+import {
+  Autocomplete,
+  BasicSpinner,
+  BasicTextInput,
+  Box,
+  Checkbox,
+  Container,
+  DemographicIndicatorNode,
+  DialogButton,
+  InputWithLabelRow,
+  ModalMode,
+  NumericTextInput,
+  Typography,
+  useDialog,
+  useKeyboardHeightAdjustment,
+  useNotification,
+  useTranslation,
+} from '@openmsupply-client/common';
+import React, { useMemo } from 'react';
+import { FC } from 'react';
+import { useVaccineCourse } from '../api/hooks/useVaccineCourse';
+import { useDemographicIndicators } from '../../IndicatorsDemographics/api/hooks/document/useDemographicIndicators';
+import { VaccineItemSelect } from './VaccineCourseItemSelect';
+import { VaccineCourseFragment } from '../api';
+
+const MAX_VACCINE_DOSES = 20;
```

Comment:
No reason now.

Originally it was so that the page looked OK with a table display, and I had imagined that 20 was beyond what any vaccine course would have in terms of doses anyway.

But I could both be wrong in that assumption, and the table formatting is no longer relevant

Reply:
I think 20 is a reasonable upper limit assumption 😅